server/src/main/java/org/solarbank/server/dto/CalculateResult.java

```diff
@@ -0,0 +1,40 @@
+package org.solarbank.server.dto;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import javax.money.CurrencyUnit;
+import javax.money.Monetary;
+import javax.money.MonetaryAmount;
+import java.util.Map;
+import lombok.Data;
+import org.javamoney.moneta.Money;
+
+@Data
+@JsonNaming(PropertyNamingStrategies.UpperCamelCaseStrategy.class)
+public class CalculateResult {
```

Comment:
added the CalculateResult class for the response from the calculate service layer